### ISSUE TITLE
Niggahīta is not normalised

### ISSUE THREAD

**Issue by mihao** [siongui/paligo#2]
Depending on the dictionary, the nasal sound (niggahīta) is transliterated as ṃ, ṁ or ŋ. For example, if you look up “ahaṃ”, you expect to see all available entries, and right now some of them are under “ahaṁ” and “ahaŋ”. Would it be possible to normalise the niggahīta as ṃ, which is the most common spelling?

**Comment by siongui** [comment]
Yes, it's possible. I have asked one Australian monk about the nasal sound transliteration and he preferred the less common spelling. I will leave this open and see if there is any good solution for different personal preferences.

**Comment by mihao** [comment]
I see, thanks for your answer. While we're at it, I noticed another similar problem. Prefixes and sufixes are marked with “-” in most dictionaries, but with “°” in the PTS dictionary. You can see this, for example, when you look up “ti-” and “ti°”. I guess it would make sense to normalise it as “-”, which is easier to type.

**Comment by siongui** [comment]
Thanks for the feedback. In summary, find a solution to normalize:

- ahaṃ ahaṁ ahaŋ
- ti- ti°

If there is anything that need to be normalized, please comment here.

**Comment by mihao** [comment]
One more case that needs to be normalised: ṅ and n. The more correct form is ṅ, but on the other hand, the use of ṅ is completely predictable: ṅ is written if, and only if, the next sound is a velar stop (k, kh, g or gh). Therefore, some dictionaries simply use n instead of ṅ. For example, look up “saṅkhāra” and “sankhāra”. The results should be merged, because it's the same word, but I think both forms should be allowed in the search box.
